components/dashboard/src/admin/License.tsx

```diff
@@ -0,0 +1,215 @@
+        })();
+    }, []);
+
+    const featureList = license?.enabledFeatures;
+    const features = license?.features;
+
+    // if user seats is 0, it means that there is user limit in the license
+    const userLimit = license?.seats == 0 ? "Unlimited" : license?.seats;
+
+    const [licenseLevel, paid, statusMessage] = license ? getSubscriptionLevel(license) : defaultMessage();
+
+    return (
+        <div>
+            <PageWithSubMenu
+                subMenu={adminMenu}
+                title="License"
+                subtitle="License associated with your Gitpod Installation"
+            >
+                <div className="flex flex-row space-x-4">
+                    <Card>
+                        {licenseLevel}
+                        {paid}
+                        <div className="mt-4 font-semibold">Available features:</div>
+                        <div className="flex flex-col items-start">
+                            {features &&
+                                features.map((feat: string) => (
+                                    <span className="inline-flex space-x-1">
+                                        {featureList?.includes(feat) ? (
+                                            <CheckSvg fill="currentColor" className="self-center" />
+                                        ) : (
+                                            <XSvg fill="currentColor" className="self-center h-2" />
+                                        )}
+                                        <span>{capitalizeInitials(feat)}</span>
+                                    </span>
+                                ))}
+                        </div>
+                    </Card>
+                    <SolidCard>
+                        <div className="text-gray-600 dark:text-gray-200 py-4 flex-row flex font-semibold items-center">
+                            {statusMessage}
+                        </div>
+                        <p className="dark:text-gray-500 font-semibold">Registered Users</p>
+                        <span className="dark:text-gray-300 pt-1 text-lg">{license?.userCount || 0}</span>
+                        <span className="dark:text-gray-500 text-gray-400 pt-1 text-lg"> / {userLimit} </span>
+                        <p className="dark:text-gray-500 pt-2 font-semibold">License Type</p>
+                        <h4 className="dark:text-gray-300 pt-1 text-lg">{capitalizeInitials(license?.type || "")}</h4>
+                        <div className="flex justify-end bottom-2 relative">
+                            <button
+                                type="button"
+                                onClick={(e) => {
+                                    e.preventDefault();
+                                    window.location.href = "https://www.gitpod.io/self-hosted";
+                                }}
+                            >
+                                <div className="font-semibold">Compare Plans</div>
```

Comment:
**issue**: Maybe it's worth opening this in a new tab since it's a link outside the dashboard. WDYT?

Eventually we could improve how we differentiate external links (see https://github.com/gitpod-io/gitpod/issues/7232) but that's a different discussion and out of the scope of this PR.

Reply:
I have changed it to be a link instead of a button according to your designs.